discoveries/coastal-flooding-and-slr.discoveries.mdx

```diff
@@ -0,0 +1,36 @@
+---
+featuredOn:
+  - eis
+id: 'eis-coastal-risks'
+name: Coastal risks from sea-level change and the changing water cycle
+description: "NASA measurements and models shed light on present and future coastal risks"
+media:
+  src: ::file ./agriculture-food-security-east-africa-cover.jpg
+  alt: Lone tree in a field in Kenya
+  author:
+    name: Damian Patkowski
+    url: https://unsplash.com/photos/T-LfvX-7IVg
+pubDate: 2022-11-06
+thematics:
+  - eis
+---
+<Block>
+   <Prose>
+    ## Sea level rise off the coast of Bangladesh
+      This chart shows the increase in sea level off the coast of Bangladesh over time.
+      
+      [PLACEHOLDER -- DENIS WILL FILL IN WITH TEXT ABOUT THE CHART]
+
+  </Prose>
+  <Figure>
+    <Chart
+      dataPath={new URL('./ECCO_and_MEaSUREs_SSH.csv', import.meta.url).href}
```

Comment:
@dfelikson The dataPath here also needs updating

Reply:
Done!